open_humans/templates/account/login-form.html

```diff
@@ -4,7 +4,9 @@
   action="{% url 'account_login' %}" id="login-form">
 
   {% csrf_token %}
-  <input type="hidden" name="next" value="{% get_next_url %}">
+
+  <input type="hidden" name="next"
```

Comment:
I guess we don't use this anymore...

Reply:
No actually keep it! see below, this helps the no-JS users. But form processing should make sure it goes into session (but not if session is already set)